src/scalper_utils.ts

```diff
@@ -307,4 +310,67 @@ export async function getFairValue(connection, spotMarket, symbol) {
     }
   }
   return fairValue;
+}
+
+export async function jupiterHedge(connection: Connection, owner: Keypair, hedgeSide: string, 
+    base: string, quote: string, hedgeDelta: number, hedgePrice: number) {
+  const jupiter = await Jupiter.load({
+    connection: connection,
+    cluster: cluster,
+    user: owner, 
+    wrapUnwrapSOL: false, 
```

Comment:
do we want this to be false?

Reply:
Conforms with OpenBook which doesn't automatically swap SOL to wSOL when we run out. The wSOL I've been managing inventory separately figured it's best to avoid unwrapping & wrapping unnecessarily

Reply:
Sounds good, consistency is preferable. If we had our way, would both be wrap and unwrap?

Reply:
Like how it is, just personal preference to have wSOL allocated purely for trading and to top up as needed